### PR TITLE
Allow text format warning to remove multiple formats.

### PR DESCRIPTION
**Pull request by Xithrius**
Closes https://github.com/python-discord/bot/issues/943

Users sometimes post `.csv` or `.json` files without being redirected to the paste system.

This PR resolves that issue by still deleting their file, but mentioning what file got deleted, which file extension it contained, and how they can access our pasting service.

- [x] Functionally works after posting text, JSON, and Comma Separated Value files.
- [x] Working written tests.

### CODE REVIEW

File: tests/bot/exts/filters/test_antimalware.py [hunk 1]
```diff
@@ -105,20 +105,35 @@ async def test_python_file_redirect_embed_description(self):
 
     async def test_txt_file_redirect_embed_description(self):
         """A message containing a .txt file should result in the correct embed."""
```
Comment (Numerlor) [inline tests/bot/exts/filters/test_antimalware.py:107]: This should be changed to include the other text like files

File: tests/bot/exts/filters/test_antimalware.py [hunk 2]
```diff
@@ -105,20 +105,35 @@ async def test_python_file_redirect_embed_description(self):
...
+                    embed.description,
+                    antimalware.TXT_EMBED_DESCRIPTION.format.return_value
+                )
+                antimalware.TXT_EMBED_DESCRIPTION.format.assert_called_with(
+                    blocked_extension_str=disallowed_extension,
+                    cmd_channel_mention=cmd_channel.mention
+                )
 
     async def test_other_disallowed_extension_embed_description(self):
         """Test the description for a non .py/.txt disallowed extension."""
```
Comment (Numerlor) [inline tests/bot/exts/filters/test_antimalware.py:139]: Same as above, should include the other extensions in some way

**Review by Numerlor — COMMENTED** [review]
Any thoughts on moving the text like extensions to a resource file and dumping a longer list of text extensions that may show up? Or do we only want to handle the ones that pop up regularly in the server.

**Review by vcokltfre — APPROVED** [review]
LGTM! Should be good when Numerlor's comments are addressed

File: bot/exts/filters/antimalware.py [hunk 3]
```diff
@@ -15,9 +15,11 @@
     f"please use a code-pasting service such as {URLs.site_schema}{URLs.site_paste}"
 )
 
+TXT_LIKE_FILES = {".txt", ".csv", ".json"}
 TXT_EMBED_DESCRIPTION = (
     "**Uh-oh!** It looks like your message got zapped by our spam filter. "
-    "We currently don't allow `.txt` attachments, so here are some tips to help you travel safely: \n\n"
+    "We currently don't allow `{blocked_extension_str}` attachments, "
```
[the remark this answers is not in the document]
Comment (Xithrius) [reply bot/exts/filters/antimalware.py]: Resolved in a305d3983350fbf30b873fce76a44707b549fd55.

File: tests/bot/exts/filters/test_antimalware.py (hunk 1 above)
Comment (Xithrius) [reply tests/bot/exts/filters/test_antimalware.py]: Resolved in aeaef8ff604c9ea62fdf1602200ee87f2adf7f6a.

File: tests/bot/exts/filters/test_antimalware.py (hunk 2 above)
Comment (Xithrius) [reply tests/bot/exts/filters/test_antimalware.py]: Resolved in aeaef8ff604c9ea62fdf1602200ee87f2adf7f6a.

**Review by Numerlor — APPROVED** [review]

**Comment by kosayoda** [comment]
I think a resource file for textfile extensions would be overkill. In reality, even `.csv` and `.json` file uploads are few and far between, so I think this solution is perfectly fine for now.

**Review by kosayoda — APPROVED** [review]
LGMT

**Pull request merged**